packages/react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -522,16 +525,10 @@ class ContextualMenuInternal extends React.Component<IContextualMenuInternalProp
     defaultRender?: IRenderFunction<IContextualMenuListProps>,
   ): JSX.Element => {
     let indexCorrection = 0;
-    const { ariaLabel, items, labelElementId, totalItemCount, hasCheckmarks, hasIcons, role } = menuListProps;
+    const { items, totalItemCount, hasCheckmarks, hasIcons } = menuListProps;
+
     return (
-      <ul
-        className={this._classNames.list}
-        aria-label={ariaLabel}
-        aria-labelledby={labelElementId}
-        onKeyDown={this._onKeyDown}
-        onKeyUp={this._onKeyUp}
-        role={role ?? 'menu'}
-      >
+      <ul className={this._classNames.list} onKeyDown={this._onKeyDown} onKeyUp={this._onKeyUp} role={'presentation'}>
```

Comment:
Is `role='presentation'` really needed here?

Reply:
This element does need `role = {‘presentation’}` since it breaks the strict parent/child relationship of a menu and menu item 
![a11ytree contextual menu](https://user-images.githubusercontent.com/8649804/113435829-701a0e80-9398-11eb-962b-26c3d8728598.JPG)